participants/ikabansal04.md

```diff
@@ -1,5 +1,22 @@
-Name: Ishika Bansal
-Github Username: ikabansal04
-University: Indira Gandhi Delhi Technical University for Women
-Year: 2
-Branch: CSE
+Name: Ishika Bansal     
+Github Username: ikabansal04   
+University: Indira Gandhi Delhi Technical University for Women    
+Year: 2       
+Branch: CSE  
```

Comment:
Giving line breaks would be really a good idea. Leaving a line or enter twice to give a line break.